### ISSUE TITLE
Data points in MMBtu are not displaying when metric units are selected

### ISSUE THREAD

**Issue by peltonc** [AASHE/bt#148]
Example: https://stars-bt-stage.herokuapp.com/my22e6

**Comment by peltonc** [comment]
Data points that are calculated fields are not displaying in either Imperial or metric units.

**Comment by dodget** [comment]
@peltonc - Would it be possible to provide a public benchmark for me to view where calculated fields are not displaying? Thanks!

**Comment by peltonc** [comment]
@dodget Here you go - Potable water use displays, but the calculated fields (e.g. Gallons/person) do not:
https://stars-bt-stage.herokuapp.com/WAb358

Public benchmarks for sharing are such a cool feature. :)

**Comment by dodget** [comment]
It is cool, and very useful for debugging. Thanks!

**Comment by peltonc** [comment]
MMBtu, MTCO2e, and US/Canadian Dollars are different from other units in STARS because they apply to all institutions, irrespective of their unit preference. So the [unit conversion](https://stars.aashe.org/_ad/credits/unit/) between metric and Imperial is 1:1.  I'm not sure if that is the root of the issue or not though.

**Comment by dodget** [comment]
Thank you for the background, very helpful. I wasn't sure if I was missing something before I dug into the problem.

**Comment by dodget** [comment]
Possibly found the culprit to second bug on line 124 of submissionvalues.py:
```
for df in credit.documentationfield_set.filter(type="numeric"):

                    etl_obj = get_df_etl_obj(df, ss)
                    if etl_obj:
                        obj_list.append(etl_obj)
```
I believe we should also be filtering for "calculated".

**Comment by dodget** [comment]
@peltonc - I believe I've cracked both bugs on this issue. Your benchmark from earlier, using calculated fields, seems to be working now (https://stars-bt-stage.herokuapp.com/WAb358). The only strange thing is that "Potable water use per weighted campus user, performance year" seems to not appear, but it is just that the figures are minuscule next to the other values being graphed.

I also checked that metric units with a 1:1 ratio with imperial units now display when the metric toggle is triggered. It looks good.

Closing now; let me know if there is an issue.
